src/XMakeTasks/AssemblyDependency/AssemblyInformation.cs

```diff
@@ -489,6 +490,9 @@ internal static string GetRuntimeVersion(string path)
             {
                 return ManagedRuntimeVersionReader.GetRuntimeVersion(path);
             }
+#else
+                return ManagedRuntimeVersionReader.GetRuntimeVersion(path);
```

Comment:
I haven't seen this code before so I'm hesitant to touch it right now.  Maybe @AndyGerlicher knows if its safe to remove that code?